students/rmart300/lesson08/circle.py

```diff
@@ -0,0 +1,72 @@
+import math
+
+class Circle(object):
+
+    def __init__(self, radius):
+        self._radius = radius
+
+    @property
+    def radius(self):
+        return self._radius
+
+    @radius.setter
+    def radius(self, radius):
+        if radius < 0:
+            self._radius = 0
+        else:
+            self._radius = radius
```

Comment:
The getter and setter don't seem necessary here for `radius`, since you could handle all this logic in the __init__ method